src/views/batch/components/batch-details/batch-details.view.jsx

```diff
@@ -13,6 +13,7 @@ import Col from '../../../shared/col/col'
 function BatchDetails ({ batch }) {
   const classes = useBatchDetailsStyles()
   const [areDeailsVisible, setDetailsVisible] = React.useState()
+  const date = new Date(batch.timestamp).toLocaleString() + ' (GMT +' + new Date().getTimezoneOffset() / -60 + ')'
```

Comment:
I think that this is not going to work on all cases, because if we are in `GMT-1` for example, the string would return `(GMT +-1)` as we have a `+` sign hardcoded in the string. Maybe checking the current timezone we can add a `+` or `-` sign manually? Or maybe there is a better way of doing it than doing this for sure hehe

Reply:
Refactored.